packages/fxa-settings/src/components/LinkExpired/index.tsx

```diff
@@ -50,14 +41,18 @@ const getTemplateValues = (linkType: LinkType) => {
       throw new Error('Invalid link type passed into LinkExpired component');
   }
   return templateValues;
-};
+}
 
-const LinkExpired = ({ linkType }: LinkExpiredProps) => {
+const LinkExpired = ({ linkType, resendLinkHandler }: LinkExpiredProps) => {
   // TODO : Metric event(s) for expired link
 
-  const resendLinkHandler = getResendLinkHandler(linkType);
   const templateValues = getTemplateValues(linkType);
-
+  const onClickReceiveNewLink = () => {
+    if (resendLinkHandler == null) {
+      throw new Error('ResendLinkHandler missing!')
```

Comment:
nit: I don't assume this is user facing, but do we want to use use lowercase `resendLinkHandler` to match the prop name?

Reply:
Yes, I don't think this would ever be user facing. It would almost certainly never make it through the testing gauntlet. I'll adjust the casing.